workbench-core/accounts/src/services/projectService.test.ts

```diff
@@ -1488,4 +1494,205 @@ describe('ProjectService', () => {
       await expect(projService.createProject(params, user)).rejects.toThrow('Failed to create project');
     });
   });
+
+  describe('softDeleteProject', () => {
+    const deletedProject1: Project = {
+      ...project1,
+      status: ProjectStatus.DELETED
+    };
+
+    const deletedProjItem1 = {
+      ...deletedProject1,
+      pk: `PROJ#${project1.id}`,
+      sk: `PROJ#${project1.id}`,
+      resourceType: 'deleted_project',
+      dependency: deletedProject1.costCenterId
+    };
+
+    let projectId: string;
+    let request: DeleteProjectRequest;
+    let checkDependency = async function (projectId: string): Promise<void> {
+      return;
+    };
+
+    beforeEach(() => {
+      request = { projectId };
+      checkDependency = async function (projectId: string): Promise<void> {
+        return;
+      };
+    });
+
+    describe('if project does not exist', () => {
+      beforeEach(() => {
+        request.projectId = 'invalid-proj-id';
+
+        // mock get project ddb call
+        const getItemResponse: GetItemCommandOutput = {
+          Item: undefined,
+          $metadata: {}
+        };
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        jest.spyOn(Getter.prototype as any, 'execute').mockImplementationOnce(() => getItemResponse);
+      });
+
+      test('it should fail', async () => {
+        // TEST n CHECK
```

Comment:
Nitpick: It's `OPERATE and CHECK`

Reply:
Will refactor! Thanks. idk why I thought it was `TEST`